File: Python/stage2/S2_FOR_02.py
# S2_FOR_02 — Find first even
def find_first_even(nums):
   for n in nums:
       if n % 2 == 0:
           return n
   return None
print(find_first_even([1, 3, 5, 8, 9]))
print(find_first_even([1, 3, 5]))

File: js/stage2/S2_FOR_02.js
// S2_FOR_02 — Find first even
function findFirstEven(nums) {
  for (const n of nums) {
    if (n % 2 === 0) {
      return n;
    }
  }
  return null;
}
// Tests
console.log(findFirstEven([1, 3, 7, 4, 9]));
console.log(findFirstEven([1, 3, 5]));
